docs/serving/architecture.md

```diff
@@ -0,0 +1,52 @@
+This page explains the high-level architecture of Knative Serving. Please also refer to [the Knative Serving Overview](./README.md) 
+and [the Request Flow](./request-flow.md) for additional information.
+
+## Diagram
+
+![Knative Serving Architecture](images/serving-architecture.png)
+
+## Components
+
+| Component      | Responsibilities                                                                                                                                                                                                                                                                                                                                                 |
+|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Activator      | The activator is part of the **data-plane**. It is responsible to queue incoming requests (if a `Knative Service` is scaled-to-zero). It communicates with the autoscaler to bring scaled-to-zero Services back up and forward the queued requests. Additional details can be found [here](https://github.com/knative/serving/blob/main/docs/scaling/SYSTEM.md). |
+| Autoscaler     | The autoscaler is responsible to scale the `Knative Services` based on configuration, metrics and incoming requests.                                                                                                                                                                                                                                             |
+| Controller     | It is the controller's job to manage the state of `Knative Resources` within the cluster. It watches several objects, manages the live cycle of dependent resources, and updates the resource state.                                                                                                                                                             |
+| Domain-Mapping | The domain-mapping controller is responsible to manage `DomainMapping` resources. It is an extension of the above `Controller`.                                                                                                                                                                                                                                  |
+| Queue-Proxy    | The Queue-Proxy lives as a `sidecar-container` aside from the `user-container` in a `Knative-Service`. It is responsible to collect metrics and making sure, requests are sent to the `user-container` in the configured rate. It can also act as a queue if necessary, similar to the Activator.                                                                |
```

Comment:
```suggestion
| Queue-Proxy    | The Queue-Proxy lives as a `sidecar-container` aside from the `user-container` in a Knative Service. It is responsible to collect metrics and making sure, requests are sent to the `user-container` in the configured rate. It can also act as a queue if necessary, similar to the Activator.                                                                |
```

Use monospace for specific processes or fields, but not for resource names, and definitely not simply for the term `Knative`.